src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheck.java

```diff
@@ -383,11 +393,17 @@ public int[] getRequiredTokens() {
     public void visitToken(DetailAST ast) {
         final Details details = Details.getDetails(ast);
         final DetailAST rcurly = details.rcurly;
+        final DetailAST semiColon = details.nextToken;
 
         if (rcurly != null) {
             final String violation = validate(details);
             if (!violation.isEmpty()) {
-                log(rcurly, violation, "}", rcurly.getColumnNo() + 1);
+                if(ast.getType() == TokenTypes.VARIABLE_DEF && details.nextToken.getType() == TokenTypes.SEMI){
+                    log(semiColon, violation, ";", rcurly.getColumnNo() + 2);
+                }
+                else {
+                    log(rcurly, violation, "}", rcurly.getColumnNo() + 1);
+                }
```

Comment:
@nrmancuso @romani

I had to modify the visitToken() method otherwise it would be impossible to log violation message for ;. If you see properly, in the master branch, rightcurly logs violation for } only. To remove this false negative, I had to modify visitToken() like that.

I am currently solving some false positive within the code, will push the changes very soon.